gammapy/image/utils.py

```diff
@@ -969,7 +939,9 @@ def cube_to_spec(cube, mask, weighting='none'):
         Summed spectrum of pixels in the mask.
     """
     value = cube.dat
-    A = solid_angle(cube)
+    from .maps import SkyMap
```

Comment:
I don't think that is the right fix to `cube_to_spec`.
Is there a test for it?

In any case, my suggestion would be that we leave that to a future PR.
Maybe add a line to the function for now saying:
```
# TODO: clean up API and implementation and add test
```
Also -- in the meantime we have introduced the `gammapy.cube` sub-package.
Could you please move the function here?
https://github.com/gammapy/gammapy/blob/master/gammapy/cube/utils.py
Then the delayed import shouldn't be necessary, i.e. you can have that import at the top of the file.

In a separate PR we'll have to discuss API / implementation for this function (e.g. whether to make it a `SkyCube` method or not), but for now let's just move it there and continue with `gammapy.image` cleanup.